### DISCUSSION TITLE
DmrVBM-v1.3.1

### DISCUSSION THREAD

**Discussion by Sandman13sq** [Sandman13sq/DmrVBM-blender-to-gms2#12] [Announcements]
- Blender Addon
	- Updated to support Blender 4.1.x
	- Implemented exporting objects by material.
	- Added Bone Dissolve panel for omitting specific bones on export.
- GMS2 Scripts
	- New VBMModel methods for managing mesh visibility.
	- Animator can blend transforms from last animation to new animation.
	- Small animation optimizations and bugfixes.
---

`DmrVBM-v1_3.zip` contains the addon and GMS2 package, along with the README containing installation instructions.

`vbm_tutorials_v1_3.zip` contains a GMS2 project file with examples of using vertex buffers and loading VBM data.

<hr /><em>This discussion was created from the release <a href='https://github.com/Sandman13sq/DmrVBM-blender-to-gms2/releases/tag/v1.3.1'>DmrVBM-v1.3.1</a>.</em>

**Comment by KeriSvinta** [comment]
Hey, I don't know if I'm being dumb or something but for some reason I don't see the add-on in blender pop up after I installed it. (This is the first time I'm using the add-on, and I have Developer Extras enabled.) Please help
![Untitled](https://github.com/Sandman13sq/DmrVBM-blender-to-gms2/assets/173104016/c1db2f89-3ef8-4007-81e8-84b9daa5baa9)

**Reply by Sandman13sq** [reply]
The addon zip file is nested in the main download. 
Make sure you install the `DmrVBM_Blender.zip` file after extracting it from the main zip download.

**Reply by KeriSvinta** [reply]
Thanks!
